KsApi/ServiceType.swift

```diff
@@ -407,6 +407,12 @@ extension ServiceType {
     headers["Authorization"] = self.authorizationHeader
     headers["Kickstarter-App-Id"] = self.appId
     headers["Kickstarter-iOS-App"] = self.buildVersion
+    headers["User-Agent"] = Self.userAgent
```

Comment:
what's the difference between `self` and `Self`? 🕵🏾‍♀️

Reply:
`Self` (capitalized) refers to the type that conforms to the protocol. (In this case, `ServiceType`). While `self` is equivalent to `this` in Java.

Reply:
amazing